Contribution guide updates (#2465)
SumoLogic/sumologic-documentation: docs/contributing/edit-doc.md

1. Push your branch to the forked repo.
1. Visit [GitHub](https://github.com/SumoLogic/sumologic-documentation) after pushing your branch. If you see an option to **Compare & pull request** for your branch, click this.<br/>![compare](/img/contributing/compare-pr.png)
   * If you don't see it, create a new [PR](https://github.com/SumoLogic/sumologic-documentation/compare).
      1. Select `main` for the base branch. This is the branch all staging and production content builds from.
      1. Select your branch for the **compare**.
      1. Click **Create Pull Request**.
1. On the Pull Request page, enter the following:
   * Make sure the **base** branch is `main` and **compare** branch is the one you pushed.
   * Enter a title for the PR.
   * If applicable, add issue number from Jira or similar program.
   * Describe your updates.
   * Apply a [label](https://github.com/SumoLogic/sumologic-documentation/wiki#github-labels) that best describes your contribution.
1. [Optional]: For urgent, high-priority PRs (for example, doc edits tied to a GA release happening within 24 hours):

>    1. Add the GA release date to the title. F or example, `AWS Integration release (GA: Jan 1, 2023)`.

Reviewer: ```suggestion
   1. Add the GA release date to the title. For example, `AWS Integration release (GA: Jan 1, 2023)`.
```
